README.md

```diff
@@ -694,6 +694,10 @@ wandb_name: # Set the name of your wandb run
 wandb_run_id: # Set the ID of your wandb run
 wandb_log_model: # "checkpoint" to log model to wandb Artifacts every `save_steps` or "end" to log only at the end of training
 
+# mlflow configuration if you're using it
+# Make sure your `MLFLOW_TRACKING_URI` is set.
+mlflow_experiment_name: # Your experiment name
```

Comment:
It might be worth logging a warning when this is set but the uri env isnt.

Reply:
Wouldn't it be better to make a config `mlflow_tracking_uri: `? The parsing code should be able to set that if needed.

Reply:
I agree with what you propose @NanoCode012 . Have update the code.